From d262ab277a87c1cda4f71137f6bb963066ba6997 Mon Sep 17 00:00:00 2001
From: Vishnu Banavath <vishnu.banavath@arm.com>
Date: Tue, 13 Sep 2022 16:46:14 +0100
Subject: [PATCH 26/27] plat: add corstone1000 platform to drivers/arm

This change is to add corstone1000 platform cmake. Smm gateway
uses SE proxy to route the calls for any NV storage.

Signed-off-by: Vishnu Banavath <vishnu.banavath@arm.com>
Upstream-Status: Pending [Not submitted to upstream yet]
---
 platform/providers/arm/corstone1000/platform.cmake | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 platform/providers/arm/corstone1000/platform.cmake

diff --git a/platform/providers/arm/corstone1000/platform.cmake b/platform/providers/arm/corstone1000/platform.cmake
new file mode 100644
index 00000000..14a9f6b0
--- /dev/null
+++ b/platform/providers/arm/corstone1000/platform.cmake
@@ -0,0 +1,12 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2021, Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Platform definition for the corstone1000 platform.
+#-------------------------------------------------------------------------------
+
+target_compile_definitions(${TGT} PRIVATE
+	SMM_GATEWAY_NV_STORE_SN="sn:ffa:46bb39d1-b4d9-45b5-88ff-040027dab249:1"
+)
+
-- 
2.17.1

